plugins/train/_config.py

```diff
@@ -7,7 +7,7 @@
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
-MASK_TYPES = ["none", "dfaker", "dfl_full"]
+MASK_TYPES = ["none", "dfaker", "dfl_full", "components"]
 MASK_INFO = "The mask to be used for training. Select none to not use a mask"
```

Comment:
Please edit info as follows (adding a description for what components does)
```
MASK_INFO = ("The mask to be used for training:"
             "\n\tnone: Don't use a mask."
             "\n\tdfaker: A basic face hull mask."
             "\n\tdfl: An improved face hull mask"
             "\n\tcomponents: <INSERT DEFINITION HERE>")
```

Reply:
Ahh, started pylinting now (10.0/10.0) but haven't flake8 yet...